ui/shared/Clipboard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClipboard } from '@fortawesome/free-regular-svg-icons';
+
+const Clipboard = ({ thing, text }) => {
+  const copyToClipboard = () => navigator.clipboard.writeText(text);
+
+  const title = `Copy ${thing}`;
+
+  return (
+    <button type="button" title={title} onClick={copyToClipboard}>
```

Comment:
I don't think this needs the `type` attribute but it does need `role`.

Reply:
Instead of using `thing` variable, could you be more descriptive i.e. 'description' or 'textTitle'? Could also move the template literal right into the `title` prop instead of assigning a variable. :)

Reply:
If I change `type` to `role`, I get:
```
error: Missing an explicit type attribute for button (react/button-has-type) at ui/shared/Clipboard.jsx:10:5:
   8 | 
   9 |   return (
> 10 |     <button
     |     ^
  11 |       role="button"
  12 |       title={`Copy ${description}`}
  13 |       onClick={copyToClipboard}


error: The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided (jsx-a11y/no-redundant-roles) at ui/shared/Clipboard.jsx:10:5:
   8 | 
   9 |   return (
> 10 |     <button
     |     ^
  11 |       role="button"
  12 |       title={`Copy ${description}`}
  13 |       onClick={copyToClipboard}
```
I've made the other changes.

Reply:
Ah right, so you do need both `type` and `role` _only_ if a span or div was still being used. `role` isn't needed for screen readers if we're using the button element.

Reply:
One more request :) I took a look at the button and it needs some style tweaks. Can you add this to the button `className="px-1"` and wrap the button in a span with `className="pr-1"`?

Reply:
Done. I also adjusted the spacing in the job-id view.